infra/modules/aws-ssm-secrets/main.tf

```diff
@@ -3,18 +3,26 @@
 #   eg, GCP Secret Manager, AWS SSM Parameter Store, Hashicorp Vault, etc.
 #  but is this good Terraform style? clearly in AWS case, this module doesn't do much ...
 
+locals {
+  # AWS SSM param name must be fully qualified if contains `/`;
+  # so test for that case, and prefix with `/` if needed
+  non_empty_path           = length(var.path) > 0
+  non_fully_qualified_path = length(regexall("/", var.path)) > 0 && !startswith(var.path, "/")
+  path_prefix              = local.non_empty_path && local.non_fully_qualified_path ? "/${var.path}" : var.path
```

Comment:
this is defensive, as ultimately added validation at the upper levels of the (eg `examples` and `modular-examples/`); but still this *should* do the right thing if someone uses our modules directly